Taskfile.yml

```
@@ -139,9 +139,10 @@ tasks:
       DEBUG: '{{.DEBUG | default "False"}}'
     cmds:
       - task: py:update
-      - pipenv run python -c \
-        'from distutils.util import strtobool; \
-        debug = bool(strtobool("{{.DEBUG}}")); \
+      - |
+        pipenv run python -c \
+        'from distutils.util import strtobool;
+        debug = bool(strtobool("{{.DEBUG}}"));
         from {{.PROJECT_SLUG}} import utils; utils.update(debug=debug)'
```

Please? For OCD's sake?

```suggestion
from {{.PROJECT_SLUG}} import utils;
utils.update(debug=debug)'
```